cluster/validate-cluster.sh

```diff
@@ -27,6 +27,16 @@ source "${KUBE_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 MINIONS_FILE=/tmp/minions-$$
 trap 'rm -rf "${MINIONS_FILE}"' EXIT
 
+# Ensure proper SED version is available.
+SED=sed
+if which gsed &>/dev/null; then
+  SED=gsed
+fi
+if ! ("$SED" --version 2>&1 | grep -q GNU); then
+  echo "!!! GNU sed is required.  If on OS X, use 'brew install gnu-sed'."
```

Comment:
For some reason, it works with the BSD sed in Yosemite. But seems to fail on Maverick (OS X 10.9). Rather than keep fighting it, this works for all... :)